### PR TITLE
HMAN-847 unNotifiedHearings - Return cancellation responses - part2

### PR DESCRIPTION
**Pull request by markdathornehmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/HMAN-847

### Change description ###

Audit this increase in version in the new audit tables

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/hmc/service/common/HearingStatusAuditService.java [hunk 1]
```diff
@@ -2,13 +2,26 @@
...
+
+    void saveAuditTriageDetailsWithUpdatedDate(HearingEntity hearingEntity, String hearingEvent,
+                                               String httpStatus,String source, String target,
+                                               JsonNode errorDescription);
+
+    void saveAuditTriageDetailsWithUpdatedDate(HearingEntity hearingEntity,
+                                String hearingEvent,String httpStatus, String source, String target,
+                                JsonNode errorDetails, JsonNode otherInfo);
+
+    HearingStatusAuditEntity getLastAuditTriageDetailsForHearingId(String hearingId);
```
[the remark this answers is not in the document]
Comment (markdathornehmcts) [reply src/main/java/uk/gov/hmcts/reform/hmc/service/common/HearingStatusAuditService.java]: Removed as no longer used

File: src/main/java/uk/gov/hmcts/reform/hmc/service/common/HearingStatusAuditServiceImpl.java [hunk 2]
```diff
@@ -25,26 +26,107 @@ public HearingStatusAuditServiceImpl(HearingStatusAuditRepository hearingStatusA
...
+                                                      JsonNode errorDetails, JsonNode otherInfo) {
+        HearingStatusAuditEntity hearingStatusAuditEntity = mapHearingStatusAuditDetailsWithUpdatedDate(hearingEntity,
+                                                                                         hearingEvent,httpStatus,
+                                                                                         source, target, errorDetails,
+                                                                                         otherInfo);
+        saveHearingStatusAudit(hearingStatusAuditEntity);
+    }
+
+    @Override
+    public HearingStatusAuditEntity getLastAuditTriageDetailsForHearingId(String hearingId) {
```
[the remark this answers is not in the document]
Comment (markdathornehmcts) [reply src/main/java/uk/gov/hmcts/reform/hmc/service/common/HearingStatusAuditServiceImpl.java]: Done

File: src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java [hunk 3]
```diff
@@ -239,10 +240,11 @@ void shouldThrowErrorForExceptionFlow() throws JsonProcessingException {
         List<ILoggingEvent> logsList = listAppender.list;
         assertEquals(3, logsList.size());
         assertEquals(Level.INFO, logsList.get(0).getLevel());
-        assertEquals("Message of type ERROR received", logsList.get(0).getMessage());
+        assertEquals("Message of type ERROR received", logsList.get(0).getFormattedMessage());
```
Comment (padmapriyanalam) [inline src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java:243]: MessageType.ERROR  can be used to be in sync with other messages.

**Review by padmapriyanalam — APPROVED** [review]
one minor comment.
LGTM

File: src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java (hunk 3 above)
Comment (markdathornehmcts) [reply src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java]: Good spot. Implemented.

**Pull request merged**